setup.cfg

```diff
@@ -73,7 +73,6 @@ addopts =
 # mypy configurations: http://bit.ly/2zEl9WI
 allow_redefinition = False
 check_untyped_defs = True
-disallow_any_explicit = True
```

Comment:
How come we remove this?

Reply:
I had thought I needed this for some typing, but it turns out I can just omit the type that's hard to represent. Added back.